packages/@aws-cdk/aws-cognito-identitypool-alpha/test/integ.identitypool.ts

```diff
@@ -58,10 +41,10 @@ const clientToImport = userPoolToImport.addClient('clientToImport');
 const importedUserPool = UserPool.fromUserPoolArn(stack, 'ImportedUserPool', userPoolToImport.userPoolArn);
 const importedUserPoolClient = UserPoolClient.fromUserPoolClientId(stack, 'ImportedUserPoolClient', clientToImport.userPoolClientId);
 const provider = new UserPoolAuthenticationProvider({ userPool, userPoolClient: client });
-const importedProvider = new UserPoolAuthenticationProvider({ userPool: importedUserPool, userPoolClient: importedUserPoolClient });
+new UserPoolAuthenticationProvider({ userPool: importedUserPool, userPoolClient: importedUserPoolClient });
 const idPool = new IdentityPool(stack, 'identitypool', {
   authenticationProviders: {
-    userPools: [provider, importedProvider],
```

Comment:
is this not allowed anymore?

Reply:
It should still be allowed, this might have been a result of a weird merge changing between branches. I'll reimplement it 👍